scripts/variantstore/wdl/GvsWarpTasks.wdl

```diff
@@ -23,9 +23,11 @@ task SNPsVariantRecalibratorCreateModel {
         File dbsnp_resource_vcf_index
         Boolean use_allele_specific_annotations
         Int max_gaussians = 6
+        Int sample_every_nth_variant = 1
```

Comment:
Is this gonna override this value being set in the filter wdl?

Reply:
This is a default because the input in GvsCreateFilterSet (which sets this value) is optional.